dips/dip-10.md

```diff
@@ -178,14 +185,24 @@ The format of the success response is:
 | status       | str | Y | Either `success` or `failure`. |
 | result       | Result object | Y | The Result obejct of response. |
 | cid           | str         | Y            | A unique identifier for the Command. Should be a UUID according to [RFC4122](https://tools.ietf.org/html/rfc4122) with "-"'s included. |
+| error          | [OffChainErrorObject](https://github.com/diem/dip/blob/main/dips/dip-1.mdx#offchainerrorobject) | N | Details of the error when status == "failure" |
+
 
 **CommandResultObject:**
 
 | Field 	    | Type 	     | Required? 	| Description 	           |
 |-------	    |------	     |-----------	|-------------	           |
 | _ObjectType   | str        | Y | Fixed value: `ReferenceIDCommandResponse`|
-| recipient_address       | str | Y | Receiver's on-chain address |
+| receiver_address       | str | Y | Receiver's on-chain address |
 
+
+#### Error Codes
+
+`duplicate_ref_id`: Duplicate Reference ID was rejected by the receiving end
+
+`invalid_user`: Receiving end could not find the user with the given user_identifier
```

Comment:
should we distinguish `invalid_receiver` and `unknown_sender` and `unknown_sender_address` and `invalid_sender_address` for when the sender address isn't the same as the txn sender or one of its child accounts?

Reply:
How would invalid_sender_address get sent back? the reference ID exchange would've ended already by the time the receiver has gotten the on-chain transaction?